packages/netlify-cms-backend-git-gateway/src/implementation.ts

```diff
@@ -251,12 +253,17 @@ export default class GitGateway implements Implementation {
     return this.authClient;
   }
 
-  requestFunction = (req: ApiRequest) =>
-    this.tokenPromise!()
+  requestFunction = (req: ApiRequest) => {
+    if (this.includeCookiesInRequests) {
+      unsentRequest.withCredentials('include', req);
```

Comment:
I believe this has not impact as `withCredentials` doesn't mutate `req`, but creates a new one.

```suggestion
      req = unsentRequest.withCredentials('include', req);
```